Proposals/xGov-187

```diff
@@ -0,0 +1,75 @@
+---
+id: 187
+period: 4
+title: AlgoMinter - NFT Image Generator
+author: Zachary Minner (@1forh)
+email: zach@skycatchfire.com
+discussions-to: https://github.com/algorandfoundation/xGov/pull/187
+company_name: minner.algo
+category: dApps
+focus_area: NFT
+open_source: Yes
+funding_type: Retroactive
+amount_requested: 25000
+delivery_date: 2024-04-06
+status: Final
+---
+
+## Abstract
+I have developed an app that allows users to generate NFT collections utilizing user-uploaded image data (layers) as well as customizations that offer fine-grain control over the generated images. 
+
+The dApp allows the user to upload layer files, set rules for specific traits, preview images based on collection size, and then download the images and ARC69 JSON data necessary to mint them on the Algorand blockchain.
+
+Here is a link to the live dApp: https://www.algominter.art
+
+## Team
+My name is Zachary Minner. I've been a Web Developer for 10+ years. I've built over 100+ websites/apps through my day job as well as over 10 different websites for Algorand NFT collections.
+
+## Experience with Algorand
+For the last 2+ years, I've been building an NFT community called Shitty Kitties with my wife Haley. Shitty Kitties is more than just NFTs. We have many utilities through our website and our Discord channel that offer users a unique and Shitty experience.
+- Wallet utilities like Mass Add and Mass Send
+- Entertainment/Rewards utilities like casino, raffles, and vending machine
+- Profile look-up 
+- and more
+
+I've also built over 12 websites/dApps for different Algorand collections as well as Discord bots, exactly to the collections/creator specifications. 
+
+I build dApps using Next.js/TypeScript and I'm very familiar with the JavaScript AlgoSDK as well as AlgoNode APIs.
+
+## Present Proposal
+The Image Generator is the main feature of the app. A user defines "Layers" (ie. Backgrounds, Hats, Accessories) for their project. For each layer, the user can upload as many images as they want to represent different "traits" for that layer.
```

Comment:
Please provide public URL of where/how the community can check proposal deliverables.